src/components/chip/index.tsx

```diff
@@ -278,7 +278,7 @@ const Chip = ({
               marginRight: Spacings.s2
             };
           } else {
-            return {marginHorizontal: Spacings.s3};
+            return {paddingHorizontal: Spacings.s3};
```

Comment:
What about the other cases? Does it work fine there?

Reply:
I couldn't see any issue with the other cases.

Reply:
Ok. Let's do that for now